submissions/Khatsur/html-css-popup/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css" />
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Hooli style Popup</title>
+</head>
+
+<body>
+  <nav class="navbar">
+    <ul class="navlist">
+      <li><a href="/">Mail</a></li>
+      <li><a href="/">Images</a></li>
+      <li id="popupicon">
+        <input type="checkbox" class="check_input" id="show" />
+        <label for="show" class="check"><img src="./images/popup-button.png" id="popupimage" alt="popupicon" /></label>
```

Comment:
Your menu button is not accessible from the keyboard. [Watch video](https://share.vidyard.com/watch/B5477pHa2Thh824MeRs4gW?) Also, avoid shifting sibling elements on hover, focus, etc.
Add `cursor: pointer` to all interactive elements.